examples/actionGroupErrors.bicep

```diff
@@ -9,7 +9,7 @@ var actionGroupEmail = 'sampleactiongroup@contoso.com'
 
 resource supportTeamActionGroup 'Microsoft.Insights/actionGroups@2021-09-01' = {
   name: actionGroupName
-  location: 'westus'
+  location: location
```

Comment:
This was an intentional error for unit testing ("Errors" suffix added for clarity). Please add a comment disabling any hard coded location check instead (and hopefully it won't break the unit test that uses `bicep build`).